src/labs_examples/lambdas/labs/exercise_03/Exercise_03.java

```diff
@@ -0,0 +1,41 @@
+package labs_examples.lambdas.labs.exercise_03;
+
+import java.util.Comparator;
+import java.util.function.BiFunction;
+import java.util.function.BinaryOperator;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+/**
+ * Lambdas Exercise 3:
+ *
+ *      1) Demonstrate the use of a static method reference
+ *      2) Demonstrate the use of an instance method reference
+ *      3) Demonstrate the use of a constructor reference
+ *
+ */
+
+class Exercise_03 {
+    public static void main(String[] args) {
+
+        Comparator<Integer> xPlusYSquared = MathFunctions::xPlusYSquared;
+        System.out.println("static method reference: " + xPlusYSquared.compare(5,5));
+
+        Exercise_03 myApp = new Exercise_03();
+        System.out.println("instance method reference: " + Exercise_03.
+                mergeThings("Hello ", "World!", myApp::appendStrings2));
```

Comment:
In Java convention we tipically put `.` on the next line:
```suggestion
        System.out.println("instance method reference: " + Exercise_03
                .mergeThings("Hello ", "World!", myApp::appendStrings2));
```
This way it is easier to notice that it is a method called on an object, rather than just methond.